avaframe/ana5Utils/distanceTimeAnalysis.py

```diff
@@ -673,6 +620,12 @@ def importMTIData(avaDir, modName, inputDir='', simHash=''):
     else:
         mtiInfoPickles = list(inputDir.glob('*%s.p' % simHash))
 
+    if len(mtiInfoPickles) == 0:
+        message = ('No mtiInfo dictionary found in %s - consider first running avalanche simulations' %
+            inputDir)
+        log.error(message)
+        raise FileNotFoundError(message)
```

Comment:
maybe use the function you just created in fU : fileNotFoundMessage

Reply:
done